test/dynamo/test_minifier.py

```diff
@@ -142,64 +86,48 @@ def inner(x):
         """
         )
 
-        test_code = self._gen_test_code(run_code, "dynamo", repro_level, backend_code)
+        test_code = self._gen_test_code(run_code, "dynamo", repro_level, "")
         proc, repro_dir = self._run_test_code(test_code)
         self.assertEqual(proc.returncode, 0)
         self.assertIsNone(repro_dir)
 
     def test_after_dynamo_cpu_compile_backend_passes(self):
-        self._test_after_dynamo_backend_passes("cpu", 2, RELU_COMPILE_ERROR_BACKEND)
+        self._test_after_dynamo_backend_passes(
+            "cpu", 2, "relu_compile_error_TESTING_ONLY"
+        )
 
     def test_after_dynamo_cpu_runtime_backend_passes(self):
-        self._test_after_dynamo_backend_passes("cpu", 2, RELU_RUNTIME_ERROR_BACKEND)
+        self._test_after_dynamo_backend_passes(
+            "cpu", 2, "relu_runtime_error_TESTING_ONLY"
+        )
 
     def test_after_dynamo_cpu_accuracy_backend_passes(self):
-        self._test_after_dynamo_backend_passes("cpu", 4, RELU_ACCURACY_ERROR_BACKEND)
+        self._test_after_dynamo_backend_passes(
+            "cpu", 4, "relu_accuracy_error_TESTING_ONLY"
+        )
 
     @requires_cuda()
     def test_after_dynamo_cuda_compile_backend_passes(self):
-        self._test_after_dynamo_backend_passes("cuda", 2, RELU_COMPILE_ERROR_BACKEND)
+        self._test_after_dynamo_backend_passes(
+            "cuda", 2, "relu_compile_error_TESTING_ONLY"
+        )
 
     @requires_cuda()
     def test_after_dynamo_cuda_runtime_backend_passes(self):
-        self._test_after_dynamo_backend_passes("cuda", 2, RELU_RUNTIME_ERROR_BACKEND)
+        self._test_after_dynamo_backend_passes(
+            "cuda", 2, "relu_runtime_error_TESTING_ONLY"
+        )
 
     @requires_cuda()
     def test_after_dynamo_cuda_accuracy_backend_passes(self):
-        self._test_after_dynamo_backend_passes("cuda", 4, RELU_ACCURACY_ERROR_BACKEND)
-
-    # Ensure that generated code with a custom backends generates a runnable minifier
-    # launcher script that results in a RuntimeError
-    def test_after_dynamo_custom_backend(self):
```

Comment:
I didn't really understand what this test was testing, that the compile time error backend isn't already testing.